VideoLocker/res/values/strings.xml

```diff
@@ -16,6 +16,8 @@
 
     <!-- Enroll Now button text-->
     <string name="enroll_now_button_text">Enroll now</string>
+    <!-- Content description of play icon that shows up with some videos-->
```

Comment:
@mdinino I'm curious about "shows up with some videos" (not all videos)?
In this comment I'd suggest clarifying that this play icon takes you to the video on the platform where it's hosted, hence "Go to video" rather than "Play video" for the content description.